packages/perseus-core/src/utils/split-perseus-item.test.ts

```diff
@@ -175,6 +175,7 @@ describe("splitPerseusItem", () => {
                             },
                         ],
                         buttonSets: ["basic"],
+                        extraKeys: ["PI"],
```

Comment:
Since ServerItemRenderer doesn't do propUpgrade now (it's expected data is upgraded by the parser outside of ServerItemRenderer), test data had to be upgraded. We could probably modify the `renderQuestion` test util to do this, but I kind of like having the data up-to-date as it shows how the JSON will actually look.